lib/ash/resource/verifiers/verify_actions_atomic.ex

```diff
@@ -71,7 +71,7 @@ defmodule Ash.Resource.Verifiers.VerifyActionsAtomic do
           dsl
           |> Ash.Resource.Info.notifiers()
           |> Enum.filter(fn notifier ->
-            notifier.requires_original_data?(module, action.name)
+            notifier.requires_original_data?(module, action)
```

Comment:
🤔 We need to keep this the same but change the type spec to match the implementation, otherwise this is a breaking change.

Reply:
Thanks for clarifying.